src/irc/ClientPool.ts

```diff
@@ -208,6 +208,10 @@ export class ClientPool {
             return bridgedClient;
         }
 
+        if (userId && await this.ircBridge.getStore().isUserDeactivated(userId)) {
```

Comment:
The check for truthiness looks a bit odd here. According to the type annotation the parameter must contain a string and the only falsey string is an emtpy string.

Reply:
good point